Source/WebKit/UIProcess/API/glib/WebKitSettings.cpp

```diff
@@ -4380,16 +4375,14 @@ gboolean webkit_settings_apply_from_key_file(WebKitSettings* settings, GKeyFile*
         }
     }
 
-    size_t length;
     GUniqueOutPtr<GError> getKeysError;
-    GUniquePtr<char*> allKeys(g_key_file_get_keys(keyFile, groupName, &length, &getKeysError.outPtr()));
+    auto allKeys = gKeyFileGetKeys(keyFile, groupName, getKeysError);
```

Comment:
Try using `Vector<CString>` here instead.

Reply:
We cannot use `Vector<T>` in these cases becase there is no way of making a `Vector`/`VectorBuffer` adopt a raw pointer coming from elsewhere. Note that GLib is allocating the memory for the container in this case so we would need to either convert (copying), or figure out a way of adopting the returned array and its contents.